src/OfflineWorkflowsSample/OfflineWorkflowsSample/Views/Controls/SceneWithTools.xaml

```diff
@@ -0,0 +1,124 @@
+        <SplitView.Pane>
+            <Pivot x:Name="OuterPivot">
+                <PivotItem Header="Info">
+                    <ScrollViewer>
+                        <StackPanel>
+                            <TextBlock Text="{x:Bind Item.Item.Title,Mode=OneWay}"
+                                       Style="{StaticResource SubtitleTextBlockStyle}" />
+                            <Image Margin="5"
+                                   Source="{x:Bind Item.Thumbnail,Mode=OneWay}" />
+                            <TextBlock Text="{x:Bind Item.Item.Description,Mode=OneWay}"
+                                       Style="{StaticResource BodyTextBlockStyle}"
+                                       TextWrapping="Wrap" />
+                            <Button Content="Open in ArcGIS Online"
+                                    HorizontalAlignment="Stretch"
+                                    Command="{Binding Source={StaticResource MainViewModel},Path=OpenItemInAgolCommand}" />
+                        </StackPanel>
+                    </ScrollViewer>
+                </PivotItem>
+                <PivotItem Header="Legend">
+                    <ScrollViewer>
+                        <ItemsControl ItemsSource="{x:Bind Scene.OperationalLayers, Mode=OneWay}"
+                                      Margin="10">
+                            <ItemsControl.ItemTemplate>
+                                <DataTemplate>
+                                    <StackPanel>
+                                        <TextBlock Text="{Binding Name}" FontWeight="Bold" />
+                                        <controls1:LayerLegend LayerContent="{Binding}" Margin="10,0" />
+                                    </StackPanel>
+                                </DataTemplate>
+                            </ItemsControl.ItemTemplate>
+                        </ItemsControl>
+                    </ScrollViewer>
+                </PivotItem>
+                <PivotItem Header="Layers">
+                    <ScrollViewer>
+                        <StackPanel>
+                            <TextBlock Text="Operational layers"
+                                       Style="{StaticResource SubtitleTextBlockStyle}" />
+                            <ListView ItemsSource="{x:Bind Scene.OperationalLayers,Mode=OneWay}"
+                                      ItemTemplate="{StaticResource LayerItemTemplate}" />
+                            <TextBlock Text="Basemap layers"
+                                       Style="{StaticResource SubtitleTextBlockStyle}" />
+                            <ListView ItemsSource="{x:Bind Scene.Basemap.BaseLayers,Mode=OneWay}"
+                                      ItemTemplate="{StaticResource LayerItemTemplate}" />
+                            <ListView ItemsSource="{x:Bind Scene.Basemap.ReferenceLayers,Mode=OneWay}"
+                                      ItemTemplate="{StaticResource LayerItemTemplate}" />
```

Comment:
Instead of showing basemap and reference layers, Just show a TextBlock with the basemap name, so it's treated as just a single entity.